README.md

```diff
@@ -31,6 +31,8 @@
       }
     }
 
-4 - Configure your GTM ID on the VTEX Admin, openning the APP's session at the left-side menu.
+## Google Tag Manager Setup
 
-See an example in [Dreamstore](https://github.com/vtex-apps/dreamstore/blob/master/pages/pages.json).
\ No newline at end of file
+To capture and analyze your store data on `GTM/Analytics`, you must create a tag associated with a trigger with the name of the event expected to listen, and publish them to the master workspace. Also, you have to go to the store administration and add the GTM ID on the `Apps` page. After the GTM configuration, you can access the `Google Analytics` and observe the event appearing on the dashboard.
```

Comment:
...you must create a tag associated with a trigger and the name of the event expected to be listenned, finally publish...